data/human/human missions.txt

```diff
@@ -6327,7 +6327,7 @@ mission "Care Package to South 1"
 					decline
 			`	You make your way through the crowd to the edge of her bubble. As soon as you do, she extends her hand. "I'm Torrey. Torrey Dupont."`
 			`	You take the hand and shake it. "<first> <last>."`
-			`	"I can tell you're independent captain. I would like to hire you to transport a package." Torrey indicates the box beside her with a nudge of her toe.`
+			`	"I can tell you're an independent captain. I would like to hire you to transport a package." Torrey indicates the box beside her with a nudge of her toe.`
```

Comment:
```suggestion
			`	"I can tell you're independent, Captain. I would like to hire you to transport a package." Torrey indicates the box beside her with a nudge of her toe.`
```

Alternatively; this is how I read the intent of the original.

As PRed is probably clearer, but figures the difference in tonal and paving were worth mention.

Reply:
I think the PRed version is more accurate. She would be looking for an independent captain (i.e. one who is not employed by either the FW or the Republic) not a captain with an independent personality.

Reply:
What is the meaning of this phrase? Are they referring to the player as "Captain", and saying they can tell they are an independent? Or are they talking to us individually and saying they can tell we are an "Independent Captain"? I'm not sure which I like better.

Reply:
I thought it meant that you were unaffiliated with a faction, so you'd be harder to track.

Reply:
To expound on the above, it's important that the package be delivered by an unaffiliated captain. If it were brought by an FW captain, it would look bad for Torrey. If it were brought by a Republic captain, it would look bad for Irene. A merchant captain would go unnoticed/unmolested, which is the goal here, since the package is actually harmless.

Reply:
"Indepdent" here is indeed still a mention of affiliation, not a character trait. It's not as common a usage, and clearly causing some confusion.

Why I said as PRed is clearer and did this as a comment just to consider against other common errors, not a request for change. :)